manager.go

```diff
@@ -558,10 +561,12 @@ func (m *Manager) InitWithOptions(elf io.ReaderAt, options Options) error {
 	}
 
 	// Patch instructions
-	if m.InstructionPatcher != nil {
-		if err := m.InstructionPatcher(m); err != nil {
-			resetManager(m)
-			return err
+	if m.InstructionPatchers != nil {
```

Comment:
Don't think you need this check, `range`-ing over `nil` is fine

Reply:
Yep, not needed. Changed, thx!